examples/wide_ep/slurm_scripts/submit.sh

```
@@ -1,20 +1,48 @@
 
 echo "Please find the \`disaggr_torch.slurm\` script in the \`examples/disaggregated/slurm/\` directory."
-echo "Make sure that SLURM parameters are correctly set in \`disaggr_torch.slurm\` before executing this script."
 
 mtp_size=0
 ntasks_per_node=4 # 4 GPUs per GB200 node
 
+isl=1024
+osl=1024
+multi_round=10
+streaming=true
+container_image=<container_image>
```

Shellcheck (0.10.0)

[error] Couldn't parse this variable assignment. Fix to allow more checks.

(SC1073)